src/main/java/seedu/trackermon/model/show/Show.java

```diff
@@ -101,9 +101,9 @@ public int hashCode() {
     public String toString() {
         final StringBuilder builder = new StringBuilder();
         builder.append(getName().toString());
-        builder.append(getStatus().toString());
-        builder.append(getComment().toString());
-        builder.append(getRating().toString());
+        builder.append("; Status: " + getStatus().toString());
+        builder.append("; Rating: " + getRating().toString());
+        builder.append("; Comment: " + getComment().toString());
```

Comment:
Good job in finally fixing the show formatting issue when a show is displayed in the result display.